frontend/src/Components/ClientProfileContent.js

```diff
@@ -306,27 +331,30 @@ class ClientProfileContent extends Component {
                 onChange={props.handleChange}
               />
               <InputComponent
-                type='date'
-                label="Child's Date of Birth"
-                value={props.childbday}
-                name='childbday'
+                type='text'
+                label="Child's Education"
+                placeholder="Type in your child's education"
+                value={props.childfname}
+                name='childfname'
                 onChange={props.handleChange}
               />
             </div>
             <div className='column'>
               <InputComponent
-                type='text'
-                label="Child's Last Name"
-                placeholder="Type in your child's last name"
-                value={props.childlname}
-                name='childlname'
+                type='number'
+                label="Child's Birth Year"
+                name='spousebday'
+                placeholder="Type in your child's birth year"
+                value={props.childbday}
                 onChange={props.handleChange}
+                min={1}
+                max={9999}
```

Comment:
- You shouldn't be able to enter in 1 for a birth year...and the max shouldn't be after this year.  I don't understand the logic of using those min and max values.  Maybe do the following: 
max={new Date().getFullYear()-20}  
Let's say they must be at least 20 to use this application this ensures that you can't enter any year after 2000.  Similarly we can do maybe:
 min={new Date().getFullYear()-100}  
Then you can't click down past 100 years ago
- Change the child's birth year input component `name=childbday`

Reply:
for a better UX, we'll make this a drop-down